code/MandrillMailer.php

```diff
@@ -584,6 +584,128 @@ protected function send($to, $from, $subject, $htmlContent,
     }
 
     /**
+     * Sends emails using specified template in mandrill.
+     * Note: not all parameters and features of mandrill->messages->sendTemplate are implimented.
+     * @param  string templateName The name of the template in mandrill.
+     * @param  array globalMergeVars associative array of merge vars.
+     * @param  string to email address to send the message.
+     * @param  string from the email address the message is from.
+     * @param  string subject subject of the email.
+     * @param  array customheaders custom headers to add to the request.
+     * @return bool success indicates result to sending the message to mantrill.
+     */
+    public function sendTemplate($templateName, $globalMergeVars, $to, $from, $subject, $customheaders)
+    {
+        // Process recipients
+        $to_array = array();
+        $to_array = $this->appendTo($to_array, $to, 'to');
+        if (isset($customheaders['Cc'])) {
+            $to_array = $this->appendTo($to_array, $customheaders['Cc'], 'cc');
+            unset($customheaders['Cc']);
+        }
+        if (isset($customheaders['Bcc'])) {
+            $to_array = $this->appendTo($to_array, $customheaders['Bcc'], 'bcc');
+            unset($customheaders['Bcc']);
+        }
+
+        // Process sender
+        $fromArray = $this->processRecipient($from);
+        $fromEmail = $fromArray['email'];
+        $fromName  = $fromArray['name'];
+
+        // Create params to send to mandrill message api
+        $default_params = array();
+
+        if (self::getDefaultParams()) {
+            $default_params = self::getDefaultParams();
+        }
+
+        // Put together the parameters.
+        $params = array_merge($default_params,
```

Comment:
I would break this down differently. e.g.

```
$params = array_merge(
    $default_params,
       array(
            "subject" => $subject,
            "from_email" => $fromEmail,
            "to" => $to_array
        )
   );
```